go/runtime/host/logger.go

```diff
@@ -129,7 +137,38 @@ func (w RuntimeLogWrapper) processLogLine(line []byte) {
 	case "ERRO":
 		rtLogger.Error(msg, kv...)
 	default:
-		w.logger.Warn("log line from runtime has no known error level set, using INFO", "log_line", string(line))
+		w.internalLogger.Warn("log line from runtime has no known error level set, using INFO", "log_line", string(line))
 		rtLogger.Info(msg, kv...)
 	}
 }
+
+// filterLogger is a logger wrapper that filters out specific keys.
```

Comment:
Maybe mention the reasoning or the types of keys that are filtered out. Could also rename it to something a bit less general. Because without looking at the code, based on the current name, I expected it to accept the field names to be filtered out.

Reply:
Ah yeah, this is a private filter only used for this specific use case. Will update.

Reply:
Made it more reusable and moved to the logging package.

Reply:
👍